ConsoleCalculator.java

```diff
@@ -0,0 +1,44 @@
+import implementations.*;
+
+import java.util.Objects;
+import java.util.Scanner;
+
+public class ConsoleCalculator {
+
+    static CalculatorInvoker calculatorInvoker = new CalculatorInvoker();
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
```

Comment:
move out the code from main to  a new file and main will contain - 
`new ConsoleCalculator.run();`